src/sentry/api/serializers/models/organization.py

```diff
@@ -353,6 +353,7 @@ def serialize(
             ),
             "avatar": avatar,
             "allowMemberProjectCreation": not obj.flags.disable_member_project_creation,
+            "allowSuperuserAccess": not obj.flags.disable_superuser_access,
```

Comment:
would customers understand what this means? can we specify that this means sentry employees?

Reply:
we can add a description in the frontend for this